docs/conf.py

```diff
@@ -104,6 +104,8 @@
                         'pfeiffer_vacuum_protocol',
                         'pfeiffer_vacuum_protocol.pfeiffer_vacuum_protocol',
                         'pfeiffer_tc400_driver',
+                        'socs.agents.hwp_supervisor.agent',
```

Comment:
I think this mock is preventing the auto-generated docs from populating in the API section. See: https://socs--487.org.readthedocs.build/en/487/agents/hwp_supervisor_agent.html#agent-api

Reply:
Yep I definitely added that as a temp fix before the supervisor agent was merged in. Thanks!

Reply:
Great, both HWP supervisor agent mocks are now removed. I don't think either was really needed anymore.